File: python/6 kyu/340.py
# https://www.codewars.com/kata/52597aa56021e91c93000cb0/train/python

# Passed

def move_zeros(lst):
    result = []
    counter = 0
    for elem in lst:
        if elem == 0:
            counter += 1
        else:
            result.append(elem)
        
    result.extend([0] * counter)
    return result

output = move_zeros([9, 0, 0, 9, 1, 2, 0, 1, 0, 1, 0, 3, 0, 1, 9, 0, 0, 0, 0, 9])
print(output)

File: nodejs/7 kyu/340.js
// https://www.codewars.com/kata/5aba780a6a176b029800041c/train/javascript

// Passed

function maxMultiple(divisor, bound) {
  let result;
  for (let num = divisor; num <= bound; num++) {
    if (num % divisor === 0) {
      result = num;
    }
  }

  return result;
}

const output = maxMultiple(37, 200);
console.log(output);